stdlib/3.4/asyncio/events.pyi

```diff
@@ -75,8 +75,12 @@ class AbstractEventLoop(metaclass=ABCMeta):
         @abstractmethod
         def create_future(self) -> Future[Any]: ...
     # Tasks methods
-    @abstractmethod
-    def create_task(self, coro: Union[Future[_T], Generator[Any, None, _T]]) -> Task[_T]: ...
+    if sys.version_info >= (3, 5):
+        @abstractmethod
+        def create_task(self, coro: Union[Awaitable[_T], Generator[Any, None, _T]]) -> Task[_T]: ...
```

Comment:
I thought the condition had to be there because there were no awaitables before python 3.5. Fixed now.